### ISSUE TITLE
[Sveltekit] ReferenceError: module is not defined

### ISSUE THREAD

**Issue by cainy-a** [metonym/svelte-highlight#149]
I cannot import this into my svelte component. It works on the REPL, however on my Sveltekit site i get the following error:
```
500 module is not defined

ReferenceError: module is not defined
    at /node_modules/highlight.js/lib/languages/bash.js?v=3f3a2059:165:1
    at instantiateModule (/home/cain/source/PersonalWebsite/node_modules/vite/dist/node/chunks/dep-994e0558.js:69063:166)
```
my code is:
```
<script>
	import Highlight  from 'svelte-highlight';
	import bash from "svelte-highlight/src/languages/bash";
	import monokaiSublime from "svelte-highlight/src/styles/monokai-sublime";
</script>

<svelte:head>
	{@html monokaiSublime}
</svelte:head>

<Highlight language={bash} code="reggie usrbg.css '\\[data-user-id=&quot;(.{17,18})&quot;\\]:not\\(img\\)' '$&,'\n'[vz-user-id=&quot;$1&quot;]:not(img)' patched.css
mv patched.css usrbg.css" />
```

**Comment by metonym** [comment]
@cainy-a  Could you try adding the following dependencies to `vite.optimizeDeps.include` in your `svelte.config.cjs`?

```diff
// svelte.config.js
const node = require("@sveltejs/adapter-node");

/** @type {import('@sveltejs/kit').Config} */
module.exports = {
  kit: {
    adapter: node(),
    target: "#svelte",
+    vite: {
+      optimizeDeps: {
+        include: [
+          "highlight.js/lib/core.js",
+          "highlight.js/lib/languages/bash.js",
+          "highlight.js/lib/languages/css.js",
+          "highlight.js/lib/languages/javascript.js",
+          "highlight.js/lib/languages/xml.js",
+        ],
+      },
+    },
  },
};

```

**Comment by cainy-a** [comment]
thanks so much that fixed it!

**Comment by metonym** [comment]
@cainy-a If you're interested, I published [version 2.0.0](https://github.com/metonym/svelte-highlight/releases/tag/v2.0.0) that uses `highlight.js@11`, which supports ESM. This means there is no need to manually specify CJS modules for vite to optimize.

I tested your example using a fresh SvelteKit skeleton template.

This is the svelte.config.js now:

```js
/** @type {import('@sveltejs/kit').Config} */
module.exports = {
	kit: {
		// hydrate the <div id="svelte"> element in src/app.html
		target: '#svelte'
	}
};

```
